src/dataflow/src/server.rs

```diff
@@ -470,6 +670,33 @@ where
                     lower.clone_from(&upper);
                 }
             }
+
+            for (id, source_ts_history) in self.render_state.ts_histories.borrow().iter() {
+                match source_ts_history {
+                    TimestampDataUpdate::RealTime(_) => continue,
```

Comment:
It would be great to comment this. I don't immediately see what this has to do with `report_frontiers` and its documentation that it sends progress information to the coordinator.